waltz-tools/src/main/java/com/wepay/waltz/tools/storage/StorageCli.java

```diff
@@ -156,21 +206,24 @@ private String getMetricsJson(String storageHost, int storagePort, String cliCon
             return partitionInfo;
         }
 
-        private void listPartitionInfo(Map<Integer, PartitionInfoSnapshot> partitionInfo) {
+        private StringBuilder getListPartitionInfo(Map<Integer, PartitionInfoSnapshot> partitionInfo, String storageHost, String storagePort) {
```

Comment:
"get" (verb) + "List" (verb), this method name seems odd. How about `formatPartitionInfo()`?